includes/class-rest.php

```
@@ -1255,6 +1268,52 @@ public function get_pagespeed_results( \WP_REST_Request $request ): \WP_REST_Res
+			if ( ! in_array( $strategy, array( 'mobile', 'desktop', '' ), true ) ) {
+				$strategy = '';
+			}
+
+			$trends = Pagespeed::get_trends();
+
+			// Filter to one URL when requested.
+			if ( ! empty( $url ) ) {
+				$key    = md5( $url );
+				$trends = array_filter(
+					$trends,
+					static function ( $k ) use ( $key, $strategy ) {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $k. Configured minimum length is 3. (undefined)

(ShortVariable)